### DISCUSSION TITLE
What's the difference between extensions "api" and "openai"?

### DISCUSSION THREAD

**Discussion by Blabter** [oobabooga/text-generation-webui#4969] [Q&A]
When I open "Extensions" tab I see two different extension: "API" and "openai". Docs mention only OpenAI api, but I can't find any information about plain "api". What is it and how it's different from OpenAI api?
![Screenshot_20231218_042605](https://github.com/oobabooga/text-generation-webui/assets/149204672/4989278f-632c-4cde-a8b6-295892d3dfa7)

**Comment by TheLounger** [comment]
The `api` checkbox and command line argument are just wrappers that enable the `openai` extension.

**Comment by dragos240** [comment]
There used to be a separate API for Ooba, but now there's only the OpenAI API, which is better long term, but means some extensions (like my own) need to migrate to the new API
